pages/workspace/inspector/inspector-issues-view.mdx

```diff
@@ -0,0 +1,7 @@
+---
+layout: 'guide'
+title: 'Inspector Issues View'
+abstract: 'How to use the Inspector Issues view'
+---
+
+import Link from '../../../components/Link';
```

Comment:
Wait where is all the content here 🤔  I see a lot more in the Vercel build: https://docs-git-inpector-issues-docs.avo-dev.app/workspace/inspector/inspector-issues-view